lib/v2/cmde/index.js

```diff
@@ -0,0 +1,41 @@
+const url = require('url');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const rootURL = 'https://www.cmde.org.cn';
+
+module.exports = async (ctx) => {
+    const cate = ctx.params.cate ?? 'xwdt/zxyw';
+    const link = `${rootURL}/${cate}/`;
+
+    const browser = await require('@/utils/puppeteer')({ stealth: true });
+    const page = await browser.newPage();
+    await page.goto(link);
```

Comment:
```suggestion
    const page = await browser.newPage();
    await page.setRequestInterception(true);
    page.on('request', (request) => {
        request.resourceType() === 'document' || request.resourceType() === 'script' ? request.continue() : request.abort();
    });
    await page.goto(link);
```
Please block unnecessary resources' request